n3fit/src/n3fit/model_trainer.py

```diff
@@ -981,20 +1047,32 @@ def hyperparametrizable(self, params):
             # endfor
 
         if self.mode_hyperopt:
+            # turn losses into arrays
+            l_hyper = np.array(l_hyper)
+            l_valid = np.array(l_valid)
+            l_exper = np.array(l_exper)
+
+            # Compute the loss over all folds for hyperopt
+            final_hyper_loss = self._hyper_loss.reduce_over_folds(l_hyper)
+
             # Hyperopt needs a dictionary with information about the losses
             # it is possible to store arbitrary information in the trial file
             # by adding it to this dictionary
             dict_out = {
                 "status": passed,
-                "loss": self._hyper_loss(
-                    fold_losses=l_hyper, pdfs_per_fold=pdfs_per_fold, experimental_models=exp_models
-                ),
+                "loss": final_hyper_loss,
                 "validation_loss": np.average(l_valid),
                 "experimental_loss": np.average(l_exper),
                 "kfold_meta": {
                     "validation_losses": l_valid,
+                    "validation_losses_phi2": np.array(trvl_phi2_per_fold),
```

Comment:
This is called `validation_losses_phi2` but I think it's computed over both validation and training datasets?

Reply:
Yes, @RoyStegeman you are right. Do you suggest any better provisory name for it (until we actually do that if we have time) ?

Reply:
I don't mind the `trvl` as you also use it in  `trvl_phi2_per_fold`, at least that doesn't risk causing the same confusion as specifying validation while it is both validation and training

Reply:
Done! Thanks!